cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -201,6 +203,8 @@ VeloxMemoryManager::VeloxMemoryManager(
   glutenAlloc_ = std::make_unique<ListenableMemoryAllocator>(allocator.get(), listener_.get());
   arrowPool_ = std::make_shared<ArrowMemoryPool>(glutenAlloc_.get());
 
+  FLAGS_velox_memory_num_shared_leaf_pools = 0;
```

Comment:
Should we move this to `VeloxInitializer` or somewhere? This seems like assigning the value repeatedly.

Reply:
OK